src/compute/src/rpc/service/monitor_service.rs

```diff
@@ -129,8 +140,13 @@ impl MonitorService for MonitorServiceImpl {
         }
 
         let time_prefix = chrono::Local::now().format("%Y-%m-%d-%H-%M-%S").to_string();
-        let file_name = format!("{}.risectl-dump-heap-prof.compute.dump\0", time_prefix,);
-        let dir = PathBuf::from(request.into_inner().get_dir());
+        let file_name = format!("{}.{}\0", time_prefix, AUTO_DUMP_MID_NAME);
```

Comment:
I guess you mean `MANUALLY_DUMP_MID_NAME`?

```suggestion
        let file_name = format!("{}.{}\0", time_prefix, MANUALLY_DUMP_MID_NAME);
```

Reply:
Besides, here I think it's unnecessary to distinguish manually dump, auto dump, and even jemalloc's interval-driven dump. How about displaying all of these heap dumps in one select-list in the WebUI?

Reply:
If we do not separate manually and auto, users will find it hard to identify a auto generated profiling bc they never know the file name or the time it was generated.

Reply:
Oh, Sorry, I should not comment it here. Yes, they should be distinguished in file names. I mean we may simply list all `.heap` files in the `list_heap_profile` interface, so that we don't need to filter the file names by checking file names (monitor_service.rs:L173-L210). Accordingly, the web UI won't have a select box for `manually dump / auto dump`. Just a minor suggestion. :)